pyFV3/stencils/ray_fast.py

```diff
@@ -18,7 +19,7 @@
 from ndsl.dsl.typing import Float, FloatField, FloatFieldK
 
 
-SDAY = 86400.0
+SDAY = Float(86400.0)
```

Comment:
If seconds per day is used frequently in the the ndsl, dycore, and physics, would it be better to define it in the ndsl constants section to ensure uniformity?

Reply:
Yes we probably should be moving that to `constant`